tests/testthat/test-authors.R

```diff
@@ -40,6 +40,7 @@ test_that("we can add an author", {
 
 test_that("we can add an author with ORCID via comment", {
 
+
```

Comment:
Can you please undo the whitespace changes? It is just commit noise. Also below. Thanks!

Reply:
Done!